abz/acousticbrainz.py

```diff
@@ -67,6 +67,12 @@ def is_processed(filepath):
     else:
         return False
 
+def get_processed_status(filepath):
+    query = """select reason from filelog where filename = ?"""
+    c = conn.cursor()
+    c.execute(query,(filepath,))
+    processed_status = c.fetchone()
+    return processed_status[0]
```

Comment:
There should be an additional blank line below here: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Done!